queries.sql

```diff
@@ -15,3 +15,30 @@ SELECT * FROM animals WHERE neutered = 'true';
 SELECT * FROM animals WHERE animal_name != 'Gabumon';
 
 SELECT * FROM animals WHERE weight_per_kg >= '10.4' AND weight_per_kg <= '17.3';
+
+
+/* Transaction Update 1 */
+UPDATE animals SET species = 'Unspecified'
+ROLLBACK;

```

Comment:
Please add queries before and after rollback to verify that the change was made.

Reply:
Thanks, changes implemented